### ISSUE TITLE
For people that were involved in previous development and discussion.

### ISSUE THREAD

**Issue by xmflsct** [xmflsct/yi-hack-1080p#6]
@maysamsh, if you get blue LED after booting and telnet password is blank, this means that you have flashed the firmware in [shadow-1/yi-hack-v3](https://github.com/shadow-1/yi-hack-v3). According to README, the upgraded version won't work with current RTSP implementation. You have to downgrade at least the home partition. Try this downgrade image [issues/#3 (comment)](https://github.com/xmflsct/yi-hack-1080p/issues/3#issuecomment-291118970).

**Comment by maysamsh** [comment]
@xmflsct I downgraded the home partition and not all I've got is a still yellow LED, [almost] instantly after the plugging the cam. Current root_fs on the cam is newer or the same version.

**Comment by xmflsct** [comment]
Newer rootfs should be fine.

During flashing process, the LED should flash yellow for around half a minute. If you insert the SD card with home_y20 in root and no yellow LED flashing, this means U-Boot does not accept this image file.

Try this one:
[home_y20.zip](https://github.com/xmflsct/yi-hack-1080p/files/910057/home_y20.zip)
Please download and test it soon. I will remove this link once you have it, as I didn't clean it up yet (still contains some old files for testing purpose) and I don't want to make it public yet.

**Comment by maysamsh** [comment]
@xmflsct I've tried it. Flashed successfully but it's still yellow LED after plugging in the cam.

**Comment by xmflsct** [comment]
Yellow is good. :) Now try telnet/FTP/RTSP and see if it works. If not, unplug the camera, insert SD card to computer and paste `/test/log.txt` here if there is any.

**Comment by maysamsh** [comment]
@xmflsct It's still yellow. does not blink at all (only two short blinks on the plug).

**Comment by xmflsct** [comment]
Have you tried to connect to it? Can you paste output of `/test/log.txt`?

**Comment by maysamsh** [comment]
@xmflsct I could not find required tools to make a serial connection yet :(.

**Comment by xmflsct** [comment]
No, I mean Telnet connect to it. And open `/test/log.txt` on SD card and paste the content here.

**Comment by maysamsh** [comment]
Contents of log.txt on the SD card:

```
Starting to log...
[INIT] Your firmware version is:
2.0.0.1A_201612051401
[INIT] Killing original init.sh script.
[good] busybox 1.16.1 is in the system.
[good] rtsp2301 is in the system.
Check for wifi configuration file...
/tmp/wpa_supplicant.conf /tmp/sd/test/wpa_supplicant.conf
Start wifi configuration...
Status for wifi configuration=127 (0 is ok)
Wifi configuration answer: 
Do network configuration (DHCP)
```

**Comment by xmflsct** [comment]
That's all? Weird that the process stuck at DHCP, which never happened to me. Try to change `DHCP` to no and your static IP settings in `yi-hack-1080p.cfg` and see if it works. If it still does not work, please paste again the `/test/log.txt`.

**Comment by maysamsh** [comment]
```
Starting to log...
[INIT] Your firmware version is:
2.0.0.1A_201612051401
[INIT] Killing original init.sh script.
[good] busybox 1.16.1 is in the system.
[good] rtsp2301 is in the system.
Check for wifi configuration file...
/tmp/wpa_supplicant.conf /tmp/sd/test/wpa_supplicant.conf
Start wifi configuration...
Status for wifi configuration=127 (0 is ok)
Wifi configuration answer: 
Do network configuration (DHCP)
```

I'm afraid it does not run the script at all and the log is for previous firmware.

**Comment by xmflsct** [comment]
Could be. You can try to delete it and see if a new one is created.

Attached is the 'older' rootfs, try to flash it before running this hack.
[rootfs_y20.zip](https://github.com/xmflsct/yi-hack-1080p/files/910139/rootfs_y20.zip)

**Comment by xmflsct** [comment]
By the way, the second log paste seems to still go for DHCP approach which is strange, if you have updated the DHCP option to no in `/test/yi-hack-1080p.cfg`.

**Comment by maysamsh** [comment]
@xmflsct after flashing the now I can telnet to the cam. It's status is still yellow LED.

**Comment by xmflsct** [comment]
Great! Have you tried all services (Telnet, FTP and RTSP)? Let me know if all these work for you.

Status LED will stay yellow, unlike in 720p version, as they have changed the way to operate LED color and I haven't been able to figure that out yet.

**Comment by maysamsh** [comment]
Yes it works, Telnet, FTP and RTSP. Now, won't work over the internet?

**Comment by xmflsct** [comment]
Nice! What do you mean by _won't work over the internet_?

**Comment by maysamsh** [comment]
Some functionality like the official app provides, accessing the stream over the internet.

**Comment by xmflsct** [comment]
Well, you can, like with VLC to stream it anywhere. I can also add a switch between modes (RTSP only and official app only). But that is not on the priority list unless lots of people would like to have it. As I said in the readme, the goal is to develop an RTSP implementation that can work alongside with the official ones.

**Comment by maysamsh** [comment]
@xmflsct Thanks. So, I cannot use the official app right now, is that correct?

**Comment by xmflsct** [comment]
Cheers. That's correct, you cannot use the official app to connect to it. All official programs on the camera are not initialized to give way to RTSP service. So it is your choice to go for either RTSP in this project, or official one in [shadow-1/yi-hack-v3](https://github.com/shadow-1/yi-hack-v3).
